build.gradle

```
@@ -35,3 +35,13 @@ dependencies {
 tasks.named('test') {
     useJUnitPlatform()
 }
+
+jar {
+    archiveBaseName.set("spring-gift-order")
+    archiveVersion.set("")
+    enabled = false
+}
+
+bootJar {
+    mainClass.set('gift.Application')
+}
```

```suggestion
bootJar {
    archiveBaseName.set("spring-gift-order")
    archiveVersion.set("")
}
```
이렇게 바꾸면 원하신대로 동작하게 됩니다.